contributing.md

```diff
@@ -134,6 +195,16 @@ Entropy source is a function that gets a piece of data about the browser a.k.a.
 Entropy sources are located in the [src/sources](src/sources) directory.
 Entropy component must be a simple JS value that can be JSON-encoded.
 
+Entropy source must meet the following requirements:
```

Comment:
An entropy source is a ..
Entropy sources must meet..

We should use a singular with an "An" article or a plural with no article, but not a mix between the two.